lib/modules/user-form-widgets/views/widget.html

```diff
@@ -1,16 +1,33 @@
 <style>
   {{data.widget.formattedContainerStyles}}
 </style>
+<script>
+	window.userFormValidation = {};
+</script>
 <div
 	class="vote-box"
 	id="{{data.widget.containerId}}"
 >
 	<h2> {{data.widget.title}} </h2>
 	<p>  {{data.widget.intro | nlbr}} </p>
-	<form method="POST" action="/modules/user-form-widgets/submit">
+	{{ apos.log (req) }}
+	<form method="POST" action="/modules/user-form-widgets/submit" class="user-form">
 	{% for field in data.widget.formFields %}
+		<input type="hidden" name="title[{{field.inputKey}}]" value="{{field.title}}" />
+
+		{% if field.validation %}
+		<script>
+			window.userFormValidation['{{field.inputKey}}'] = "{{ field.validation }}";
```

Comment:
To avoid the window variable you can also do this in the index.js and then set a data attribute on the form.
`widget.html`
data-form-validation="{{ data.widget.userFormValidation }}"
`index.js`
```self.load = (req, widgets, callback) => {
widgets.forEach((widget) => {
widgets.userFormValidation = mappedValidationObject;
});
}```
see data property in `public/js/main.js`
```
self.play = function($widget, data, options) {
```

Reply:
Da's een mooie! Ik heb hem nu op een soortgelijke manier opgelost, kun je hiermee leven? :)